samples/wifi/radio_test/Kconfig.sysbuild

```diff
@@ -0,0 +1,40 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+config NRF700X_RADIO_TEST_COMBO
+	bool "Enable radio test combo build for Wi-Fi and Bluetooth"
+	depends on SOC_NRF5340_CPUAPP
+
+menu "Network core configuration"
+	depends on SUPPORT_NETCORE
+
+config SUPPORT_NETCORE_PERIPHERAL_RADIO_TEST
+	bool
+	default y if NRF700X_RADIO_TEST_COMBO
```

Comment:
default y, the if isn't needed as support netcore is NRF5340 soc series (though includes NRF54H20 too)